### PR TITLE
Remove mention of "apt-get clean" as it's performed automatically

### PR DESCRIPTION
**Pull request by tianon**
Also, added a "Note" with a link to where this is configured for the official images.

### CODE REVIEW

**Comment by jfrazelle** [comment]
LGTM

**Comment by thaJeztah** [comment]
Oh, didn't know that!

File: docs/articles/dockerfile_best-practices.md [hunk 1]
```diff
@@ -230,6 +229,8 @@ keep the image size down. Since the `RUN` statement starts with
 `apt-get update`, the package cache will always be refreshed prior to
 `apt-get install`.
 
+**Note**: The official Debian and Ubuntu images [automatically run `apt-get clean`](https://github.com/docker/docker/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105), so explicit invocation is not required.
```
Comment (thaJeztah) [inline docs/articles/dockerfile_best-practices.md]: Sorry for nit-picking, but can you put this in our "standard" format, i.e.

```Markdown
> **Note**: The official Debian and Ubuntu images [automatically run `apt-get clean`](https://github.com/docker/docker/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105),
> so explicit invocation is not required.
```

**Comment by thaJeztah** [comment]
One nit, but LGTM otherwise :heart:

File: docs/articles/dockerfile_best-practices.md (hunk 1 above)
Comment (tianon) [inline docs/articles/dockerfile_best-practices.md]: Absolutely! :+1: (done)

**Comment by thaJeztah** [comment]
Awesome! Merging :hamster:

**Pull request merged**